apps/teams-test-app/src/components/privateApis/FilesAPIs.tsx

```diff
@@ -164,6 +164,59 @@ const CopyMoveFiles = (): ReactElement =>
     },
   });
 
+enum FileDownloadStatus {
+  Downloaded = 'Downloaded',
+  Downloading = 'Downloading',
+  Failed = 'Failed',
+}
+interface IFileItem {
+  objectId?: string;
+  path?: string;
+  sizeInBytes?: number;
+  status?: FileDownloadStatus;
+  timestamp: Date;
+  title: string;
+  extension: string;
+}
+
+const GetFileDownloads = (): ReactElement =>
+  ApiWithTextInput<IFileItem[]>({
```

Comment:
It looks like we won't take input from the user in this case, maybe we should use ApiWithoutInput instead?